userspace/libsinsp/filter/ast.cpp

```diff
@@ -171,6 +171,25 @@ const std::string& string_visitor::as_string()
 	return m_str;
 }
 
+libsinsp::filter::ast::expr::expr()
+{
+}
+
+libsinsp::filter::ast::expr::expr(const pos_info& pos)
+	: m_pos(pos)
+{
+}
+
+const libsinsp::filter::ast::pos_info& libsinsp::filter::ast::expr::get_pos()
```

Comment:
Nit: we might as well inline `get_pos` and `set_pos` in ast.h right away.

Reply:
This is another thing where a style guide would be useful--some devs want only declarations in header files. We don't have good rules on explicit inlining either. But if we want to allow trivial or nearly-trivial declarations, sure. I'll move them. (I'm going to move expr::get_pos()/expr::set_pos() as well then).